spring-cloud-gcp-starters/pom.xml

```diff
@@ -24,8 +24,9 @@
 		<module>spring-cloud-gcp-starter-pubsub</module>
 		<module>spring-cloud-gcp-starter-config</module>
 		<module>spring-cloud-gcp-starter-storage</module>
-		<module>spring-cloud-gcp-starter-sql</module>
 		<module>spring-cloud-gcp-starter-trace</module>
 		<module>spring-cloud-gcp-starter-logging</module>
-	</modules>
+		<module>spring-cloud-gcp-starter-sql-mysql</module>
+        <module>spring-cloud-gcp-starter-sql-postgresql</module>
+    </modules>
```

Comment:
As I've already indicated, I am -1 on a starter per database.

Reply:
We don't expect to have more in the near future. It's just a convenience for users and a better alternative to the combined `spring-cloud-gcp-starter-sql`. It doesn't have to translate into a direct entry in the Initializr.

Reply:
Duly noted. For the record, my remark has nothing to do with Spring Initializr.